src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -504,7 +504,14 @@ static std::string parseNotification(ConnectionInfo* ciP, SubscriptionUpdate* su
 
   if (notification.HasMember("attrs"))
   {
-    if (parseStringVector(&subsP->notification.attributes, notification["attrs"], "attrs", true, true, &errorString) == false)
```

Comment:
Please try to minimize changes not related with the PR, e.g. refactoring in a file which nothing has to do with the PR. A single chande doesn't hurt, but the accumulation of many of them makes review process harder.

(General comment, no only for this PR. In fact, in this PR probably doesn't worth the time to rollbacking the changes, so NTC)

Reply:
I got 4 errors from "make style" from travis.
3 about too long lines and one about space before newline.
I fixed the four of them.

NTC

Reply:
That's weird... as that case should be already in master and travis didn't complain about it.

In this PR we are not going to do anything about it (thus, I agree, NTC) but next time it happens, please tell me (by skype). I'd like to have a closer look.

Reply:
I think I know what happened.
The tree errors about "line too long" have "always" been there, but they're minor.
Now I added a MAYOR error, "space before newline" and travis shows all errors (minor and mayor) from the file. Thus, I got four errors from travis (really from 'make style').
I fixed not only the MAYOR error but also the three MINor errors.

Reply:
Ok. Mistery solved ;)